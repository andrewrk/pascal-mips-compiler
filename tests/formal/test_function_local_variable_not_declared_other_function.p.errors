ERROR: line 10: constructor for main class "testFunctionLocalVariableNotDeclaredOtherFunction" must have no parameters
ERROR: line 20: variable "someVar" not declared
