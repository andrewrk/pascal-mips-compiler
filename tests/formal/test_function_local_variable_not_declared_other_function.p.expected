ERROR: line 20: variable "someVar" not declared
